instrumentation/opentelemetry-instrumentation-aws-lambda/tests/test_aws_lambda_instrumentation_manual.py

```diff
@@ -262,6 +268,23 @@ def custom_event_context_extractor(lambda_event):
                 expected_state_value=MOCK_W3C_TRACE_STATE_VALUE,
                 xray_traceid=MOCK_XRAY_TRACE_CONTEXT_SAMPLED,
             ),
+            TestCase(
+                name="baggae_propagation",
```

Comment:
Appreciate the added tests. Super trivial suggestion:

```suggestion
                name="baggage_propagation",
```

Reply:
Fixed, thank you.